Bottom: ad996136b34f064552931b388ab3e6372d13e35f
Top:    7d81e7bd5b0cb84b248d3c3c561988f9875be8de
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:20:12 -0500

Refresh of use-docker-named-networks

---
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index 76a8a4ba..704b51d4 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -165,7 +165,7 @@
       - '{{ openldap_container_name }}:gitlab-ldap'
       - '{{ gitlab_redis_container_name }}:gitlab-redis'
       - '{{ gitlab_db_container_name }}:gitlab-db'
-      - '{{ exim4_container_name }}:{{ container_addr_map.exim4.hostname }}'
+      - '{{ exim4_container_name }}:{{ exim4_hostname }}'
     name: '{{ gitlab_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 07c24383..fe575575 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -140,13 +140,13 @@
       WIKI_MAIL_PASSWORD: '{{ wiki_email_password | quote }}'
     hostname: '{{ wiki_hostname }}'
     image: '{{ wiki_image_name }}:{{ docker_image_tag }}'
-    link:
-      - '{{ openldap_container_name }}:wiki_ldap'
-      - '{{ wiki_db_container_name }}:wiki_db'
-      - '{{ exim4_container_name }}:{{ container_addr_map.exim4.hostname }}'
     name: '{{ wiki_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        links:
+          - '{{ openldap_container_name }}:wiki_ldap'
+          - '{{ wiki_db_container_name }}:wiki_db'
+          - '{{ exim4_container_name }}:{{ exim4_hostname }}'
     ports: '{{ wiki_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 7a7dd834..5422d3e5 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -110,7 +110,7 @@
   docker_container:
     detach: true
     env:
-      LDAP_HOSTNAME: '{{ container_addr_map.ldap.hostname }}'
+      LDAP_HOSTNAME: '{{ openldap_hostname }}'
     hostname: '{{ openldap_hostname }}'
     image: '{{ openldap_image_name }}:{{ docker_image_tag }}'
     name: '{{ openldap_container_name }}'
